### PR TITLE
Some README edits

### PR DESCRIPTION
**Pull request by RichardLitt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,84 +19,96 @@ retrieve those files!
...
+- [Installation](#installation)
+- [Using via Storj CLI](#start-using-storj-via-the-uplink-cli)
 - [Using via AWS S3 CLI](#start-using-storj-via-the-aws-s3-cli)
 - [License](#license)
 - [Support](#support)
 
-# Start Contributing to Storj
+# Contributing to Storj
+
+At the moment, all of our code here for v3 is open source. Have a code change you think would make Storj better? We are definitely open to pull requests. Send them along!
```
Comment (jtolds) [inline README.md]:
```suggestion
All of our code for Storj v3 is open source. Have a code change you think would make Storj better? Please send a pull request along!
```

File: README.md [hunk 2]
```diff
@@ -19,84 +19,96 @@ retrieve those files!
...
+
+Have comments, bug reports, or suggestions? Want to propose a PR before hand-crafting it? Jump on to [our Rocketchat](https://community.storj.io) to join the community and to talk to the Storj core team.
```
Comment (jtolds) [inline README.md]:
```suggestion
Have comments, bug reports, or suggestions? Want to propose a PR before hand-crafting it? Jump on to [our Rocketchat](https://community.storj.io) and join the #dev channel to join the developer community and to talk to the Storj core team.
```

File: README.md [hunk 3]
```diff
@@ -19,84 +19,96 @@ retrieve those files!
...
 
-Install git and golang. We support Linux, Mac, and Windows operating
-systems. Other operating systems supported by Go are probably not much
-additional work.
+We support Linux, Mac, and Windows operating systems. Other operating systems supported by Go should also be able to run Storj.
 
 ### Download and compile Storj
 
-> **Aside about GOPATH**: Go 1.11 supports a new feature called Go modules,
+> **Aside about GOPATH**:  If you don't have a GOPATH set, you can ignore this > aside. Go 1.11 supports a new feature called Go modules,
```
[the remark this answers is not in the document]
Comment (RichardLitt) [reply README.md]: I didn't remove any content; I just moved this sentence to the front.

[the remark this answers is not in the document]
Comment (RichardLitt) [reply README.md]: Ah! Let's remove this sentence, then. Good catch.

**Pull request merged**